addons/hr_org_chart/static/src/xml/hr_org_chart.xml

```diff
@@ -10,13 +10,13 @@
                 use bg-images to get a clean and centred images -->
             <a t-if="! is_self"
                 class="o_media_object rounded-circle o_employee_redirect"
-                t-att-style="'background-image:url(\'/web/image/hr.employee.public/' + employee.id + '/image_1024/\')'"
+                t-att-style="'background-image:url(\'/web/image/hr.employee.public/' + employee.id + '/avatar_1024/\')'"
```

Comment:
Isn't this broken right now? See earlier comment to rename those fields in public employee model.

Reply:
I renamed the fields in hr.employee.public to avatar_xxxx. So it won't be broken anymore. good catch